app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = list()
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
```

Comment:
You can also use triple quotes for multi line comments to make it more elegant :)

```
def __repr__(self) -> str:
        return f"""
        {{Name: {self.name}, 
        Health: {self.health},
        Hidden: {self.hidden}}}
        """